BasicSample/Assets/HandTracking/Prefabs/HandJointPrefab.prefab.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 2fa7a793d3dd19542ae479953c204898
+guid: a2539ffc80d35c0458d3ad7d1ab409de
```

Comment:
interesting that the meta files are considered moved while the prefabs themselves are not by git diff.

Reply:
this is github's choice, not git.  They are doing the heuristic of file tracking on individual file's content, and it has no knowledge of unity's file convention that "file and file.meta" must be changed together.

Reply:
it gives less diff overall when meta files are moved instead of new.  I don't have control over that and I don't think they matters to the change since the guid is new here.

Reply:
Good to know, was not aware. And right, i did observe the new guid in there but was curious to know if there is something more for github to consider this a move.